### PR TITLE
i#2157 re-attach: fix the mac build broken by PR 2762

### PR DESCRIPTION
**Pull request by Carrotman42**
Issue: #2157

### CODE REVIEW

**Comment by Carrotman42** [comment]
run aarch64 tests

File: core/unix/ksynch_macos.c [hunk 1]
```diff
@@ -41,6 +41,7 @@
  */
 
 #include "../globals.h"
+#include <errno.h>
```
Comment (derekbruening) [inline core/unix/ksynch_macos.c]: Hmm it may be safest to #undef errno after including this to ensure we have no refs to it

File: .travis.yml [hunk 2]
```diff
@@ -86,7 +86,7 @@ jobs:
       env: DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY=no EXTRA_ARGS=32_only
       # XXX i#2764: Travis OSX resources are over-subscribed and it can take
       # hours to get an OSX machine, so we skip running PR's for now.
-      if: type = push
+      # if: type = push
```
Comment (derekbruening) [inline .travis.yml]: (NOCHECKIN of course -- good practice to mark it with a comment so it's clearly temporary (we used to use #NOCHECKIN but CI now yells at that so something else))

Comment (Carrotman42) [reply .travis.yml]: Ack

File: core/unix/ksynch_macos.c (hunk 1 above)
Comment (Carrotman42) [reply core/unix/ksynch_macos.c]: Done.

**Pull request merged**